src/strategy/spid.ts

```diff
@@ -59,8 +59,8 @@ export class SpidStrategy extends SamlStrategy {
   ) {
     super(options, verify);
     if (!options.requestIdExpirationPeriodMs) {
-      // 1 hour
-      options.requestIdExpirationPeriodMs = 3600 * 1000;
+      // 5 minutes
```

Comment:
after a quick chat with @pp-ps we agreed to put this to 15 minutes

Reply:
Fixed in 7e7ff8e